lib/moment/juttle-moment.js

```diff
@@ -844,6 +788,66 @@ var JuttleMoment = Base.extend({
         }
 
         return operand.valueOf();
+    },
+    momentString: function(moment) {
+        if (isNaN(moment.valueOf())) {
+            return moment._i.toString();
+        } else {
+            return moment.toDate().toISOString();
+        }
+    },
+    simpleDurationString: function(d) {
```

Comment:
That seems totally sensible to me as well -- do you know if we have an issue tracking it in the old Jira? I don't see anything in github.